server.js

```diff
@@ -93,20 +93,21 @@ var PATH = 'http://api.openweathermap.org/data/2.5/weather';
          }
          else if (trimmed.match(len1Re)) {
             mainObj.zip = -1;
+	    //pretty sure this wont work since city isn't defined yet...
```

Comment:
`cityDB` is a global from above, but `city` is not defined yet unless i'm missing something (not getting any hits on it until below this conditional branch). Not sure exactly what the intention is and/or how to correctly change this so I will leave that up in the air.